internal/namespaces/instance/v1/custom_image.go

```diff
@@ -58,6 +62,14 @@ func imageCreateBuilder(c *core.Command) *core.Command {
 // imageListBuilder list the images for a given organization.
 // A call to GetServer(..) with the ID contained in Image.FromServer retrieves more information about the server.
 func imageListBuilder(c *core.Command) *core.Command {
+	type customListImageRequest struct {
+		*instance.ListImagesRequest
+		OrganizationID *string
+	}
+
+	c.ArgSpecs.GetByName(oldOrganizationFieldName).Name = newOrganizationFieldName
```

Comment:
nit: this line is duplicated in a lot of places, we could add a small function `renameOrganizationIDArgSpec(c.ArgSpecs)`